File: app/src/main/java/br/com/wcisang/alojinha/ui/customview/PacificoTextView.java
package br.com.wcisang.alojinha.ui.customview;

import android.content.Context;
import android.graphics.Typeface;
import android.util.AttributeSet;
import android.view.Gravity;

/**
 * Created by WCisang on 07/06/2018.
 */
public class PacificoTextView extends android.support.v7.widget.AppCompatTextView {


    public PacificoTextView(Context context) {
        super(context);
        initView();
    }

    public PacificoTextView(Context context, AttributeSet attrs) {
        super(context, attrs);
        initView();
    }

    public PacificoTextView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
        initView();
    }

    private void initView() {
        setGravity(Gravity.CENTER);
        formatText();
    }

    private void formatText(){
        setTypeface(Typeface.createFromAsset(getContext().getAssets(),"Pacifico-Regular.ttf"));
    }
}

File: app/src/main/java/br/com/b2w/lodjinha/views/PacificoTextView.kt
package br.com.b2w.lodjinha.views

import android.content.Context
import android.graphics.Typeface
import android.util.AttributeSet
import android.widget.TextView

class PacificoTextView @JvmOverloads constructor(
    context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
) : TextView(context, attrs, defStyleAttr) {

    init {
        val font = Typeface.createFromAsset(context.assets, "fonts/Pacifico-Regular.ttf")
        setTypeface(font, Typeface.NORMAL)
    }

}